component/site/controllers/icalevent.php

```diff
@@ -136,14 +136,14 @@ function edit($key = NULL, $urlVar = NULL){
 		$user = JFactory::getUser();
 		if ((!$is_event_creator && !$is_event_editor) || ($user->id==0 && JRequest::getInt("evid",0)>0)){
 			if ($user->id){
-				$this->setRedirect(JURI::root(),JText::_('JEV_NOTAUTH_CREATE_EVENT'));
+				$this->setRedirect(JURI::root(), JText::_('JEV_NOTAUTH_CREATE_EVENT'), 'ERROR');
```

Comment:
Trivial point - messagetype is a lower case string so you should use 'error'.  But the Joomla application convers to lower case so its not critical